### PR TITLE
root case - snap installation

### PR DESCRIPTION
**Pull request by hypeitnow**
The case in which you have chosen to install the snap package as root user(e.g when running default lxd container, hosting juju client only)

### CODE REVIEW

File: jhack/config.py [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
-HOME_DIR = Path("/home") / os.environ["USER"]
-JHACK_DATA_PATH = HOME_DIR / ".config" / "jhack"
-JHACK_CONFIG_PATH = JHACK_DATA_PATH / "config.toml"
+
+if USR == "root":
+    HOME_DIR = "/root"
+else: 
+    HOME_DIR = Path("/home") / os.environ["USER"]
+
+JHACK_CONFIG_PATH = HOME_DIR / ".jhack/config.toml"
```
Comment (PietroPasotti) [inline jhack/config.py:18]: I'd decided to move the jhack config to /home/.config/jhack/ as per the xdg home spec https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

so this code was correct:
```
JHACK_DATA_PATH = HOME_DIR / ".config" / "jhack"
JHACK_CONFIG_PATH = JHACK_DATA_PATH / "config.toml"
```
if you can revert this change I'm fine with accepting this patch

**Review by PietroPasotti — CHANGES_REQUESTED** [review]

File: jhack/config.py (hunk 1 above)
Comment (hypeitnow) [reply jhack/config.py]: Done

**Comment by PietroPasotti** [comment]
Lovely! Thank you
